migrations/20200702152554-user_sample_csv_upload.js

```diff
@@ -0,0 +1,68 @@
+
+const fs = require('fs');
+
+const https = require('https');
+const request = require('request');
+
+module.exports = {
+  async up(db) {
+
+    global.migrationMsg = "Upload bulk user sample csv to cloud"
+   
+    let uploadfileInfo = { name: "users.csv", path: process.env.BULK_USER_SAMPLE_CSV_PATH };
```

Comment:
@rakeshSgr - We should upload sample file in git at path - {{PROJECT_ROOT}}/public/bulkUploadSamples/users.csv